writer.go

```diff
@@ -185,6 +185,9 @@ type Writer struct {
 	// If nil, DefaultTransport is used.
 	Transport RoundTripper
 
+	// AllowAutoTopicCreation notifies writer to create topic is missing.
```

Comment:
While it's not a breaking change per-say since the code will continue to compile, the new behavior we introduce is the inverse of what the program used to do.

Do you think we could add a bit more documentation to reflect that fact?

Reply:
@achille-roussel hey! indeed, you are right wrt docu update. As for failing test, I'd say that the new field shall be inversion of `AllowAutoTopicCreation` for backwards compatibility for default behaviour. It'd also "fix" the test as a side effect :) WDYT?